app/javascript/packs/cms.ts

```diff
@@ -304,21 +279,28 @@ function setUpPjax () {
 }
 
 /**
- * The CMS features 3 dropdown buttons: New button, preview button and save button. Html is rendered
- * by CMS::DropdownHelper and its functionality is given by this method everytime a new template is
+ * The CMS features 4 dropdown buttons: New button, Preview, Publish and Save. Html is rendered
+ * by "/provider/admin/cms/dropdown" partial and its functionality is given by this method everytime a new template is
  * selected (cms-sidebar:update event).
  *
  * The event handler has to be defined with jQuery because at the time this method is called, the
  * CMS content is not yet rendered and document.querySelector would find nothing.
  */
 function setUpDropdownButtonOpen () {
-  $(document).on('click', '.dropdown-toggle', (event) => {
-    closeAllDropdowns(event.target as HTMLButtonElement)
-
-    $(event.currentTarget).siblings('.dropdown').toggleClass('expanded')
-
-    return false
-  })
+  document.querySelectorAll<HTMLButtonElement>('.dropdown-toggle')
+    .forEach(toggle => {
+      toggle.addEventListener('click', () => {
+        closeAllDropdowns(toggle)
+
+        /* eslint-disable @typescript-eslint/non-nullable-type-assertion-style */
```

Comment:
@josemigallas Do we need to have this rule enabled? :thinking:  I don't see why it might be needed.
It kind of suggest (from what I can understand) to use `!` instead of type cast, but then I am having the following rule violation: https://typescript-eslint.io/rules/non-nullable-type-assertion-style/